packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/AddBankYapily/Connect/template.success.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
+
+import {
+  BankWaitIndicator,
+  BankWrapper,
+  Hr,
+  LinkOptionsWrapper,
+  LinkViaDesktop,
+  ModalNavWithCloseIcon,
+  ScanWithPhone,
+  Section
+} from '../../../../components'
+import { OwnProps as _O, Props as _P, SuccessStateType as _SS } from '.'
+
+type Props = _O & _SS & _P
```

Comment:
I think we do not need aliases here at least for `OwnProps` and `SuccessStateType `

Reply:
Carry over habit from working on exchange. I'll change it.